RealtimeJMSToStream/README.rst

```diff
@@ -0,0 +1,90 @@
+===============================================================================
+Realtime Java Message Service (JMS) Source To Stream Sink Adapter Configuration
+===============================================================================
+
+The ETL Realtime Template can be used to create an Adapter that reads from a Realtime Source and pass it to a Sink to be processed further.
+
+In this example, we will poll messages from Java Message Service (JMS) source in realtime. and use a Stream Sink to write the messages to CDAP Stream.
+
+The config.json contains a sample Adapter configuration that you can use to accomplish the above task. 
+
+Our sample Adapter uses these components:
+
+- ETLRealtime Application Template, since we want to perform ETL in realtime
+- JMS source, using the implementation of JMS from Apache ActiveMQ, version 5.11.1
+- Stream sink to hold the incoming messages to be processed by a downstream program such as a CDAP Flow
+
+You can create and start the Adapter by using the CDAP CLI (or you can use the UI for a more visual approach).
+
+The realtime JMS source uses generic JMS interface to communicate with JMS broker, so the source requires implementation of the JMS interfaces
```

Comment:
The realtime JMS source uses a generic JMS interface to communicate with a JMS broker, so the source requires that the JMS interfaces be implemented, in order to function as a separate Adapter plugin.